dlrover/go/operator/config/manifests/bases/default-role.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Remove the replicated file https://github.com/intelligent-machine-learning/dlrover/blob/master/dlrover/go/operator/config/rbac/default_role.yaml